### PR TITLE
GLideN64: store and load config version

### PR DESCRIPTION
**Pull request by gizmo98**
https://github.com/RetroPie/RetroPie-Setup/issues/1925

### CODE REVIEW

File: scriptmodules/emulators/mupen64plus/mupen64plus.sh [hunk 1]
```diff
@@ -231,7 +231,11 @@ function testCompatibility() {
             fi
             iniConfig " = " "" "$config"
             # Settings version. Don't touch it.
-            iniSet "configVersion" "17"
+            local config_version="17"
+            if [[ -f $configdir/n64/GLideN64_config_version.ini ]]; then
```
Comment (joolswills) [inline scriptmodules/emulators/mupen64plus/mupen64plus.sh]: Please use md_conf_root and quote it. Thanks.

**Comment by joolswills** [comment]
Maybe also it should just be .cfg or .txt if it's not actually an ini file. Cheers

**Comment by robertybob** [comment]
@joolswills sorry to hijack this, but the forum appears to be redirecting to http://lmgtfy.com/?q=retropie+documentation 

(I'll delete this post once you've seen it)

**Comment by joolswills** [comment]
Check the date.

File: scriptmodules/emulators/mupen64plus/mupen64plus.sh (hunk 1 above)
Comment (gizmo98) [inline scriptmodules/emulators/mupen64plus/mupen64plus.sh]: $md_conf_root should not be available in startup scripts. $configdir is declared at the beginning of the file.

Comment (joolswills) [inline scriptmodules/emulators/mupen64plus/mupen64plus.sh]: my apologies - I misread (was doing the review on my mobile ;-) )

Please can you still quote it and squash and I will merge.

Comment (gizmo98) [inline scriptmodules/emulators/mupen64plus/mupen64plus.sh]: Done!

**Comment by joolswills** [comment]
You missed the quoting on the next usage too - but ill fix once merged.

**Comment by gizmo98** [comment]
Thanks!

**Comment by joolswills** [comment]
Thanks for the PR.

**Pull request merged**